### PR TITLE
Specify grafana packages as peer- and dev deps

### PR DESCRIPTION
**Pull request by sunker**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,19 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "@emotion/css": "11.1.3",
-    "react-select": "5.2.1"
+    "react-select": "5.2.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "@grafana/data": "canary",
```
Comment (sunker) [inline package.json:28]: Would it make sense to point to `latest` now?

File: package.json [hunk 2]
```diff
@@ -22,11 +22,19 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "@emotion/css": "11.1.3",
-    "react-select": "5.2.1"
+    "react-select": "5.2.1",
```
Comment (jackw) [inline package.json:25]: I think this needs to be a semver range like `^5.2.0` which will support up to `6.0.0` and prevent cli noise when running `yarn` in projects that consume this package.

File: package.json (hunk 1 above)
Comment (jackw) [reply package.json:28]: Hmm... `latest` might work but generally with peerDependencies they set a semver range. I think something like `^9.2.0` should work here for `@grafana/` packages. It'll also help prevent noise in projects that consume this package.

**Review by jackw — APPROVED** [review]
LGTM! 🚀

**Pull request merged**